packages/combo-box/package.json

```diff
@@ -31,6 +31,7 @@
     "polymer"
   ],
   "dependencies": {
+    "@open-wc/dedupe-mixin": "^1.3.0",
```

Comment:
For now we only use `Constructor` type but we might want to also use `dedupeMixin` helper later.
It does pretty much the same thing as `dedupingMixin` from Polymer but in a different way.